src/cljs/nnts2/note/components.cljs

```diff
@@ -10,30 +10,26 @@
         title-text (r/atom (:title @note-form))
```

Comment:
title-text and content-text atoms can be collapsed into one.

Reply:
will do in the note branch